src/styles/utilities/display.ts

```diff
@@ -20,15 +19,35 @@ export default {
         display: 'none',
     },
 
+    /**
+     * Web-only style.
```

Comment:
Are we maintaining react-native-web specific types in a single file or are these types spread across multiple files?

Reply:
We have web styles spread across multiple files (at least in display.ts, positioning.ts)